pkg/apis/config/defaults.go

```diff
@@ -70,34 +72,42 @@ func NewDefaultsConfigFromConfigMap(config *corev1.ConfigMap) (*Defaults, error)
 type Defaults struct {
 	// NamespaceDefaultsConfig are the default Broker Configs for each namespace.
 	// Namespace is the key, the value is the KReference to the config.
-	NamespaceDefaultsConfig map[string]*ClassAndKRef `json:"namespaceDefaults,omitempty"`
+	NamespaceDefaultsConfig map[string]*ClassAndBrokerConfig `json:"namespaceDefaults,omitempty"`
 
 	// ClusterDefaultBrokerConfig is the default broker config for all the namespaces that
 	// are not in NamespaceDefaultBrokerConfigs.
-	ClusterDefault *ClassAndKRef `json:"clusterDefault,omitempty"`
+	ClusterDefault *ClassAndBrokerConfig `json:"clusterDefault,omitempty"`
+}
+
+// ClassAndBrokerConfig contains configuration for a given namespace for broker. Allows
+// configuring the Class of the Broker, the reference to the
+// config it should use and it's delivery.
+type ClassAndBrokerConfig struct {
+	BrokerClass   string `json:"brokerClass,omitempty"`
+	*BrokerConfig `json:",inline"`
 }
 
-// ClassAndKRef contains configuration for a given namespace for broker. Allows
-// configuring both the Class of the Broker as well as the reference to the
-// config it should use
-type ClassAndKRef struct {
-	BrokerClass        string `json:"brokerClass,omitempty"`
+// BrokerConfig contains configuration for a given namespace for broker. Allows
+// configuring the reference to the
+// config it should use and it's delivery.
+type BrokerConfig struct {
 	*duckv1.KReference `json:",inline"`
+	Delivery           *eventingduckv1.DeliverySpec `json:"delivery,omitempty"`
```

Comment:
I have questions about this place.
If I use v1Delivery, I need to convert v1Delivery to v1beta1Delivery. Should I do this? Or do you have any suggestions